### DISCUSSION TITLE
that-happens

### DISCUSSION THREAD

**Discussion by giscus** [sangwoo-joh/reactions#12] [Comments]
# that-happens

그럴 수도 있지.

https://sangwoo-joh.github.io/that-happens

**Comment by kkeundotnet** [comment]
흥미로운 글입니다. "그럴 수도 있지", 나와 다른 생각을 받아들이는 유연한 자세를 취한다는 건 생각보다 중요한데 생각보다 어렵고, 의외로 우리나라 교육과정에선 생각보다 배우거나 접할 기회가 없는 것 같습니다. 이것도 아마 같은 맥락에서 이해되는 경험이 아닐까 싶습니다.

> 사람은 정말로 감정적이고 사회적이어서, 면전에 대고 “당신은 틀렸다”고 말하는 것은, 설령 정말로 그 주제에 대해서 그 사람이 틀렸을지라 하더라도, 거의 항상 좋지 않은 결과를 가져온다는 것을 많이 경험했다.

물론 아 다르고 어 다르다고 어감을 정확하게 가져가야 하겠지만 저는 요즘 직장에서 "틀렸다"-류의 말을 자주 하려고 하는 편입니다. 일단 그런 말을 하기 전에 저 스스로 문제를 최대한 정확히 이해하려고 노력합니다. 이건 이것 자체로 저에게 이득이죠. 똑똑한(?) 사람들의 반응은 주로 두 가지였는데,

* "앗! 나는 거기까지는 생각이 미치지 못했는데! 감사감사!"
  * "난 바보인가봐!" (보통 본인의 똑똑함을 어필할 필요조차 없는 사람들이 주로 이런 반응.)
* "오호~ 좋은 질문이야! 하지만 반대로 아마 너가 놓친 포인트가 있는 것 같은데 이게 이러이러해서..."

어떤 결과든 모두에게 생산적인 방향으로 이야기가 흘러갔습니다. 제가 좋은 사람들과 함께 일하고 있다는 뜻이겠죠? 🤭

> 나도 모르게 날이 선 채 그건 틀렸다고 얘기하기 일쑤였다. ... 그런데 이게 모두에게 아무런 이득이 없다는 것을 점차 깨달았다.

아마도 이건 "나도 모르게 날이 선 채" 때문이 아니었을까요? "그럴 수도 있지"라는 말을 하시면서 시간을 벌고 계시는 것일 수도. 아무튼 좋은 방법같아요.

한편, 좋아하는 성향과 싫어하는 성향이 명확한 건 잘못이 없는 것 같습니다. 오히려 멋지고 좋은 것 같아요. 😎 물론 이거야말로 저의 선호이니 "그럴 수 있지" 생각하시면 됩니다.

**Reply by sangwoo-joh** [reply]
그렇습니다. "유연한 자세"에 대해서는 정말 오랜 세월 다양한 곳에서 중요하다는 것을 익히 들어왔는데, 스스로 실천하는 일이 정말 어렵다는 게 더 실감이 됩니다. 뭐든 그렇겠지만 생각만 하는 거랑 직접 실천하는 거랑은 아주 큰 차이가 있네요.

> 물론 아 다르고 어 다르다고 어감을 정확하게 가져가야 하겠지만 저는 요즘 직장에서 "틀렸다"-류의 말을 자주 하려고 하는 편입니다. 일단 그런 말을 하기 전에 저 스스로 문제를 최대한 정확히 이해하려고 노력합니다. 이건 이것 자체로 저에게 이득이죠. 똑똑한(?) 사람들의 반응은 주로 두 가지였는데 ....

여러가지 의미로 좋은 환경이라 부럽습니다 :sob: 저도 모든 사람에게 이런 원칙을 고수하려는 것은 아니고, 상황에 맞게 물 흘러가듯 맞춰보면서 이렇게도 해보고 저렇게도 해보고 있습니다. 제 생각을 간직한 채로 다른 사람의 의견에 부드럽게 대처를 잘 할 수 있으면 좋겠네요.
